src/i_winmusic.c

```diff
@@ -1238,6 +1252,22 @@ static void FillBuffer(void)
         return;
     }
 
+    if (player_state == STATE_STOPPED)
+    {
+        StopSound();
+        StreamOut();
+        player_state = STATE_PAUSED;
```

Comment:
Why going from stopped to paused? Add some comments.

Reply:
https://github.com/chocolate-doom/chocolate-doom/pull/1603/commits/574373bd68ba62cb4c42d4c7c3f28ac45edadba8